File: 프로그래머스/unrated/181887. 홀수 vs 짝수/홀수 vs 짝수.java
class Solution {
    public int solution(int[] num_list) {
        int answer = 0;
        int odd = 0;
        int even = 0;
        
        for (int i = 0; i < num_list.length; i++) {
            if (i % 2 == 0) {
                odd += num_list[i];
            } else {
                even += num_list[i];
            }
        }
        
        answer = (int) Math.max(odd, even);
        
        return answer;
    }
}

File: 프로그래머스/0/181887. 홀수 vs 짝수/홀수 vs 짝수.py
def solution(num_list):
    answer = 0
    tmp = 0
    
    for i in range(len(num_list)):
        if (i + 1) % 2 != 0:
            tmp += num_list[i]
        else:
            answer += num_list[i]
            
    answer = max(tmp, answer)
    
    return answer